tests/feature/test_outline.py

```diff
@@ -171,8 +171,7 @@ def test_outline_with_escaped_pipes(pytester):
     pytester.makefile(
         ".feature",
         outline=textwrap.dedent(
-            r"""
-            Feature: Outline With Special characters
+            r"""Feature: Outline With Special characters
```

Comment:
was this change necessary? it should pass with the original version too

Reply:
aaaah I see the issue now, it's a raw string, so ok nevermind we can keep it as it is